test/Microsoft.IdentityModel.Tokens.Tests/SecurityKeyTests.cs

```diff
@@ -198,7 +198,7 @@ public static TheoryData<SecurityKeyTheoryData> CreateInternalIdsTestCases
                         ExpectedInternalId = Base64UrlEncoder.Encode(KeyingMaterial.JsonWebKeyRsa_2048_Public.ComputeJwkThumbprint()),
                         TestId = nameof(KeyingMaterial.JsonWebKeyRsa_2048_Public)
                     },
-#if NET_CORE
+#if NET472 || NET_CORE
```

Comment:
nit: comment on ln216 is no longer true.

Reply:
Good catch. Fixed!